feature-libs/user/profile/components/update-email/update-email-component.service.ts

```diff
@@ -34,12 +35,15 @@ export class UpdateEmailComponentService {
 
   protected busy$ = new BehaviorSubject(false);
 
+  updateSucceed$ = new Subject();
+
   isUpdating$ = this.busy$.pipe(
     tap((state) => (state === true ? this.form.disable() : this.form.enable()))
   );
 
   form: UntypedFormGroup = new UntypedFormGroup(
     {
+      oldEmail: new UntypedFormControl(''),
```

Comment:
[QUESTION]
1. Perhaps I'm missing something obvious. From UX perspective, could you help me understand the purpose of asking an user to type an old email into separate field, when they just want to set a new one? 
2. I have problem to find a place in the code, where `newEmail` would be actually used and sent to the backend. I can see in the [lines 67-73](https://github.com/SAP/spartacus/pull/18348/files#diff-e04ca982ffc07e608054f16f121c89b564ca8f2751209619be2ff70d0e3d01cdR67-R73) that we pass to the `UserEmailFacade.update()` only the `password` and the `newEmail`. Was it deliberate? This makes me even more confused about our addition of the form field `oldEmail` in the v2 component.

Reply:
Q1.It's just show old email(read only) similar as show customerId when update profile, The end user don't need type it again. I have a video recording pasted in https://github.com/SAP/spartacus/pull/17839 and UX/PO are happy with it.

Q2,   newEmail and password is enough, as the token present who is the people try to update it's email.

Hope it's clear

Reply:
A pic is more helpful for answer Q1
![image](https://github.com/SAP/spartacus/assets/38693320/e528821b-42b6-4452-9345-c3a02bf6941a)

Reply:
[MEDIUM]
If the `oldEmail` is not used for the purpose of the form (i.e. it's not submitted to the backend, nor validated), but only for informing the user, IMHO it's a bit misleading that it's a part of the _form model_ preapared in the component-service.

Instead, I'd suggest to remove it entirely from the `this.form` model in component-service (and therefore not bind it in the html temlate via `[ngModel]`), but in the HTML rather create a standalone dummy input for it with a simple binding for the attribute `value`:
```diff
   <input
-    required="true"
     type="text"
     class="form-control"
-    name="oldEmail"
-    formControlName="oldEmail"
-    [ngModel]="(user$ | async)?.uid"
+    [value]="(user$ | async)?.uid"
     readonly
   />
```

What do you think about it?